sbor/src/rust.rs

```diff
@@ -1,3 +1,35 @@
+pub mod prelude {
+    // See eg https://doc.rust-lang.org/std/prelude/index.html
+
+    // std::prelude::v1
+    pub use super::borrow::ToOwned;
+    pub use super::boxed::Box;
+    pub use super::clone::Clone;
+    pub use super::cmp::{Eq, Ord, PartialEq, PartialOrd};
+    pub use super::convert::{AsMut, AsRef, From, Into};
+    pub use super::default::Default;
+    pub use super::iter::{DoubleEndedIterator, ExactSizeIterator, Extend, IntoIterator, Iterator};
+    pub use super::marker::{Copy, Send, Sized, Sync, Unpin};
+    pub use super::mem::drop;
+    pub use super::ops::{Drop, Fn, FnMut, FnOnce};
+    pub use super::option::Option::{self, None, Some};
+    pub use super::result::Result::{self, Err, Ok};
+    pub use super::string::{String, ToString};
+    pub use super::vec::Vec;
+
+    // std::prelude::rust_2021
+    pub use super::convert::{TryFrom, TryInto};
+    pub use super::iter::FromIterator;
+
+    // And some extra useful additions we use a lot:
+    pub use super::collections::*;
+    pub use super::fmt::{Debug, Display};
+    pub use super::format;
+    pub use super::marker::PhantomData;
+    pub use super::str::FromStr;
+    pub use super::vec;
+}
```

Comment:
This is nice!

Reply:
Hopefully will save my sanity a little xD